modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -1,68 +1,201 @@
+
+For details, see xref:policies-landing-page.adoc[Policies].
+
+=== SLA Tiers
+Service Level Access (SLA) tiers are categories of user access that you define for an API. The tier definition combined with an SLA-based policy determines whether access to the API at a certain level requires your approval. The tier definition also can limit the number of requests an application can make to the API. To enforce SLA tiers, you need to apply a rate-limiting or throttling policy that is SLA-based.
+
+For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
+
+=== Client Providers (Identity Providers)
+You use client providers to help you enforce security and regulations in your business organization. Client providers authorize client applications. 
+
+For details, see xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers for Client Management].
 
-// talk about naming, instances
+=== Contracts
+Contracts are created between client applications and API instances. A client application requests access in Exchange. The owner of the API instance approves the request in API Manager (or the approval is automatic, depending on configuration). When the request has been approved, the client application is registered.
 
-After configuring the API, you can perform API management tasks, applying policies and setting up SLA tiers, assigning permissions to the API environment,
-versioning, viewing API analytics, and more.
+Contracts are enforced with either of the following:
 
-The API dashboard shows key information about a deployed API instance:
+* SLA enforcement policies
+* Client enforcement policies
+
+For details, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
+
+=== API Alerts
+An API alert (different from a Runtime Manager alert) is an alarm that flags one of the following:
+
+* The API request violates a policy
+* Requests received by the API exceed a given number within a period of time
+* The API returns a specified HTTP error code
+* The API response time exceeds a certain amount
+
+API Manager triggers alerts when states change from desirable to undesirable or vice versa. When an alert triggers, API Manager sends email notifications to you and to others, if configured.
+
+For details, see xref:using-api-alerts.adoc[Alerts].
+
+=== API Assets
+API assets, published by Exchange, are the components that comprise applications. Components include any of the following:
+
+* APIs (OAS, RAML, RAML fragments, HTTP, WSDL, and Custom)
+* API Groups
+* Policies
+* Examples
+* Templates
+* Modules
+* Connectors owned by MuleSoft or your organization
+
+For details, see xref:exchange::about-sharing-assets.adoc[Share Assets].
+
+=== API Groups
+An API Group is an API asset that enables organizations to publish a group of API instances as a single unit, so that applications can access the APIs as a group using one client ID. API Groups are created in API Manager and published to Exchange.
```

Comment:
```suggestion
An API group is an API asset that enables organizations to publish a group of API instances as a single unit, so that applications can access the APIs as a group using one client ID. API groups are created in API Manager and published to Exchange.
```

Reply:
Fixed.